[2024-07-10T10:07:23.693+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-10T10:07:23.710+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: market_etl.create_fact_table manual__2024-07-10T10:06:44.581370+00:00 [queued]>
[2024-07-10T10:07:23.716+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: market_etl.create_fact_table manual__2024-07-10T10:06:44.581370+00:00 [queued]>
[2024-07-10T10:07:23.716+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 4
[2024-07-10T10:07:23.724+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): create_fact_table> on 2024-07-10 10:06:44.581370+00:00
[2024-07-10T10:07:23.731+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=139) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-10T10:07:23.732+0000] {standard_task_runner.py:63} INFO - Started process 145 to run task
[2024-07-10T10:07:23.733+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'market_etl', 'create_fact_table', 'manual__2024-07-10T10:06:44.581370+00:00', '--job-id', '691', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpa3q_9wc7']
[2024-07-10T10:07:23.735+0000] {standard_task_runner.py:91} INFO - Job 691: Subtask create_fact_table
[2024-07-10T10:07:23.795+0000] {task_command.py:426} INFO - Running <TaskInstance: market_etl.create_fact_table manual__2024-07-10T10:06:44.581370+00:00 [running]> on host 514b016b59c3
[2024-07-10T10:07:24.084+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='market_etl' AIRFLOW_CTX_TASK_ID='create_fact_table' AIRFLOW_CTX_EXECUTION_DATE='2024-07-10T10:06:44.581370+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-10T10:06:44.581370+00:00'
[2024-07-10T10:07:24.087+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-10T10:07:24.096+0000] {python.py:237} INFO - Done. Returned value was:     SECURITY_ID  ...                                           PRICE_ID
0          ADBL  ...  ADBL_<bound method Timestamp.date of Timestamp...
1           AHL  ...  AHL_<bound method Timestamp.date of Timestamp(...
2          AHPC  ...  AHPC_<bound method Timestamp.date of Timestamp...
3         AKJCL  ...  AKJCL_<bound method Timestamp.date of Timestam...
4          AKPL  ...  AKPL_<bound method Timestamp.date of Timestamp...
..          ...  ...                                                ...
310        USHL  ...  USHL_<bound method Timestamp.date of Timestamp...
311        USLB  ...  USLB_<bound method Timestamp.date of Timestamp...
312        VLBS  ...  VLBS_<bound method Timestamp.date of Timestamp...
313       VLUCL  ...  VLUCL_<bound method Timestamp.date of Timestam...
314        WNLB  ...  WNLB_<bound method Timestamp.date of Timestamp...

[315 rows x 9 columns]
[2024-07-10T10:07:24.105+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-10T10:07:24.154+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=market_etl, task_id=create_fact_table, run_id=manual__2024-07-10T10:06:44.581370+00:00, execution_date=20240710T100644, start_date=20240710T100723, end_date=20240710T100724
[2024-07-10T10:07:24.202+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-10T10:07:24.226+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-10T10:07:24.227+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
